apps/src/lib/cli.rs

```diff
@@ -2439,33 +2440,42 @@ pub mod args {
 
     #[derive(Clone, Debug)]
     pub struct JoinNetwork {
-        pub chain_id: ChainId,
+        pub chain_id: Option<ChainId>,
+        pub release_archive: Option<PathBuf>,
         pub genesis_validator: Option<String>,
         pub pre_genesis_path: Option<PathBuf>,
         pub dont_prefetch_wasm: bool,
     }
 
     impl Args for JoinNetwork {
         fn parse(matches: &ArgMatches) -> Self {
-            let chain_id = CHAIN_ID.parse(matches);
+            let chain_id = CHAIN_ID_OPT.parse(matches);
+            let release_archive = RELEASE_ARCHIVE.parse(matches);
             let genesis_validator = GENESIS_VALIDATOR.parse(matches);
             let pre_genesis_path = PRE_GENESIS_PATH.parse(matches);
             let dont_prefetch_wasm = DONT_PREFETCH_WASM.parse(matches);
             Self {
                 chain_id,
+                release_archive,
                 genesis_validator,
                 pre_genesis_path,
                 dont_prefetch_wasm,
             }
         }
 
         fn def(app: App) -> App {
-            app.arg(CHAIN_ID.def().about("The chain ID. The chain must be known in the https://github.com/heliaxdev/anoma-network-config repository."))
+            app.arg(CHAIN_ID_OPT.def().about("The chain ID. The chain must be known in the https://github.com/heliaxdev/anoma-network-config repository.").conflicts_with(RELEASE_ARCHIVE.name))
+                .arg(RELEASE_ARCHIVE.def().about("The path to a release archive (.tar.gz)."))
```

Comment:
Referring to the `.tar.gz` you download from https://github.com/heliaxdev/anoma-network-config/releases as a "release archive", but maybe there is a better term